pkg/volume/glusterfs/glusterfs_uid.go

```diff
@@ -0,0 +1,151 @@
+
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+
+/*
+
+Reference#
+https://github.com/openshift/origin/blob/master/pkg/security/uid/uid.go
+
+
+*/
+package glusterfs
```

Comment:
I agree that this should be made public.
And as a separate, prereq patch.
see idea about a natural api above.